tests/conftest.py

```diff
@@ -1,11 +0,0 @@
-import os
-
-import pytest
-
-
-@pytest.fixture(autouse=True)
-def environment():
-    """Test should be performed on staging environment."""
-    os.environ['JINA_HUBBLE_REGISTRY'] = 'https://apihubble.staging.jina.ai'
-    yield
-    del os.environ['JINA_HUBBLE_REGISTRY']
```

Comment:
We need to add `JINA_HUBBLE_REGISTRY` to `workflows/release.yml` as well if we remove this.

Reply:
added it